package.json

```diff
@@ -66,6 +66,9 @@
     "jpm": "^1.1.1",
     "load-grunt-tasks": "^3.5.0",
     "napa": "^2.3.0",
+    "ng-annotate": "^1.2.1",
+    "ng-annotate-loader": "^0.2.0",
+    "nginject-loader": "^2.1.0",
```

Comment:
Yup I'm pretty sure we've taken care of all the ngInject/ngAnnotate stuff. If there's something that wasn't working before, let's figure out how to make it work.